models/group.go

```diff
@@ -222,6 +223,20 @@ func insertTargetIntoGroup(t Target, gid int64) error {
 	return nil
 }
 
+// UpdateTarget updates the given target information in the database.
+func UpdateTarget(target Target) error {
+	targetInfo := map[string]interface{}{
```

Comment:
Does this need to be a `map[string]interface`? can it just be a `models.Target` instance?